crates/frontend/src/pages/context_override.rs

```diff
@@ -1,77 +1,206 @@
+                .await
+            };
+
+            match result {
+                Ok(_) => {
+                    logging::log!("Context and overrides submitted successfully");
+                    handle_submit.call(());
+                }
+                Err(e) => {
+                    logging::log!("Error submitting context and overrides: {:?}", e);
+                }
+            }
+        });
+    };
+    view! {
+        <ContextForm
+            dimensions=dimensions.get_value()
+            context=context.get_untracked()
+            is_standalone=false
+            handle_change=move |new_context| {
+                set_context
+                    .update(|value| {
+                        *value = new_context;
+                    });
+            }
+
+            disabled=edit
+        />
+        <OverrideForm
+            overrides=overrides.get_untracked()
+            default_config=default_config
+            is_standalone=false
+            handle_change=move |new_overrides| {
+                set_overrides
+                    .update(|value| {
+                        *value = new_overrides;
+                    });
+            }
+        />
+
+        <div class="form-control grid w-full justify-end">
+            <Button
+                class="pl-[70px] pr-[70px]".to_string()
+                text="Submit".to_string()
+                on_click=on_submit
+            />
+        </div>
+    }
+}
+
 #[component]
 pub fn context_override() -> impl IntoView {
     let tenant_rs = use_context::<ReadSignal<String>>().unwrap();
 
-    let selected_context_and_override = create_rw_signal::<Option<TableData>>(None);
-    let form_data = create_rw_signal::<Option<TableData>>(None);
+    let (selected_data, set_selected_data) = create_signal::<Option<Data>>(None);
+    let (form_mode, set_form_mode) = create_signal::<Option<FormMode>>(None);
+
+    let page_resource: Resource<String, PageResource> = create_blocking_resource(
+        move || tenant_rs.get().clone(),
+        |current_tenant| async move {
+            let (config_result, dimensions_result, default_config_result) = join!(
+                fetch_config(current_tenant.to_string()),
+                fetch_dimensions(current_tenant.to_string()),
+                fetch_default_config(current_tenant.to_string())
+            );
+            PageResource {
+                config: config_result.unwrap_or_default(),
+                dimensions: dimensions_result
+                    .unwrap_or_default()
+                    .into_iter()
+                    .filter(|d| d.dimension != "variantIds")
+                    .collect(),
+                default_config: default_config_result.unwrap_or_default(),
+            }
+        },
+    );
 
-    let table_columns = create_memo(move |_| {
-        vec![
-            Column::default("KEY".to_string()),
-            Column::default("VALUE".to_string()),
-        ]
+    let handle_context_create = Callback::new(move |_| {
```

Comment:
@ShubhranshuSanjeev after creating, updating overrides , when drawer closes  latest data is not showing , we have to refresh the page then data gets updated